examples/push_challenges.py

```diff
@@ -1,175 +1,146 @@
-
-#     with open(json_filename) as f:
-#         data = json.load(f)
-#         patient_bundles = data['patient_bundles']
-#         # patient_bundles = patient_bundles[:1]
-
-#     for patient_bundle in patient_bundles:
-#         patient = patient_bundle['patient']
-
-#         try:
-#             # Create a FHIR Patient
-#             patient = patient_api.create_patient(
-#                 dataset_id,
-#                 fhir_store_id,
-#                 patient=patient)
-#             print(f"patient: {patient}")
-
-#             # Create the Note and Annotation objects linked to the patient
-#             note_bundles = patient_bundle['note_bundles']
-#             # note_bundles = note_bundles[:1]
-
-#             for note_bundle in note_bundles:
-#                 note = note_bundle['note']
-#                 note['patientId'] = patient.id
-#                 try:
-#                     note = note_api.create_note(
-#                         dataset_id,
-#                         fhir_store_id,
-#                         note=note
-#                     )
-#                     print(f"note: {note}")
-#                 except ApiException as e:
-#                     print("Exception when calling NoteApi->create_note: %s\n" % e)
-
-#                 annotation = note_bundle['annotation']
-#                 annotation['annotationSource']['resourceSource']['name'] = \
-#                     "{fhir_store_name}/fhir/Note/{note_id}".format(
-#                         fhir_store_name=fhir_store.name,
-#                         note_id=note.id
-#                     )
-#                 try:
-#                     annotation = annotation_api.create_annotation(
-#                         dataset_id,
-#                         annotation_store_id,
-#                         annotation=annotation
-#                     )
-#                     print(f"annotation: {annotation}")
-#                 except ApiException as e:
-#                     print("Exception when calling AnnotationApi->create_annotation: %s\n" % e)
-
-#                 # time.sleep(1)
-
-#         except ApiException as e:
-#             print("Exception when calling PatientApi->create_patient: %s\n" % e)
-
-#         # time.sleep(1)
+
+def reformat_id(s):
+    """Reformat ID string.
+
+    Valid IDs must be lowercase and single dash-delimited.
+    """
+    s = s.replace("'", "")
+    s = re.sub(r"[ \"'\.?!/]", "-", s)
+    s = re.sub(r"\-+", "-", s)
+    return s.lower()
+
+
+def add_tag(api, tag):
+    """Create a new Tag entry in the database.
+
+    Exception:
+        409: tagId already exists
+    """
+    tag_id = reformat_id(tag.get("tagId"))
+    try:
+        api.create_tag(tag_id, tag=roccclient.Tag())
+    except ApiException as err:
+        if err.status == 409:
+            print(f"Duplicate tag: {tag_id}")
+        else:
+            print(f"Exception when calling TagApi->create_tag: {err}")
+
+
+def add_org(api, org):
```

Comment:
Replace by `add_organization`
Prefer the full name - more descriptive - unless you want to save space, e.g. to fit in the 80-char line limit

Reply:
Nope! I was just being lazy :laughing: